game/match/match.go

```diff
@@ -698,6 +703,44 @@ func (m *Match) newMultipartActionBase(player *Player, cards map[string][]*Card,
 
 }
 
+// NewDrawCountAction prompts the user to select a number of cards to draw
+func (m *Match) NewDrawCountAction(player *Player, text string, minSelections int, maxSelections int) {
```

Comment:
This can probably be used for other situations than just drawing so maybe we should rename it to not mention drawing?

Reply:
Changed